### PR TITLE
Durcissement II: Behind the scenes

### PR DESCRIPTION
**Pull request by wouter-admiraal-sonarsource**
See https://github.com/SonarSource/sonar-enterprise/pull/2821

### CODE REVIEW

**Review by jeremy-davis-sonarsource — APPROVED** [review]
LGTM!

**Review by siegfried-ehret-sonarsource — APPROVED** [review]

**Comment by wouter-admiraal-sonarsource** [comment]
Sorry, took the liberty of adding a 2nd commit. Could you have a look plz 🙏 ?

**Review by siegfried-ehret-sonarsource — APPROVED** [review]

File: components/controls/buttons.css [hunk 1]
```diff
@@ -281,6 +281,10 @@
   color: #fff;
 }
 
+.button.button-icon.disabled {
+  background: transparent !important;
+}
```
Comment (jeremy-davis-sonarsource) [inline components/controls/buttons.css:286]: Looks good, but the icon becomes white on hover, which means invisible in many cases. I'd make the icon always grey, but maybe it's better to add the background back on hover?

Comment (wouter-admiraal-sonarsource) [reply components/controls/buttons.css:286]: Oh, good catch. I think keeping it gray makes more sense. Why have a hover state if it's disabled in the first place? Hover state suggests interactivity.

**Pull request merged**